pom.xml

```diff
@@ -5,7 +5,7 @@
 
     <groupId>fr.insee</groupId>
     <artifactId>public-enemy-api</artifactId>
-    <version>2.2.6</version>
+    <version>2.2.7</version>
```

Comment:
I'd have bumped the minor version for a change that include a new version of java (but that's a detail)

Reply:
2.3.0 ?